client/webserver/site/src/html/register.tmpl

```diff
@@ -26,12 +26,12 @@
 
     {{- /* Set up the Decred wallet. Only shown if user has not connected their Decred wallet. */ -}}
     <form class="card mx-auto my-5 bg1{{if not .WalletStep}} d-hide{{end}}" id="newWalletForm">
-      {{template "newWalletForm" "Your Decred wallet is required to pay registration fees."}}
+      {{template "newWalletForm" .}}
```

Comment:
I didn't notice this before, but rendering these forms when called from the register page caused an error because they could not find `.UserInfo.IsPasswordCached`.`